doc/documentation_data_format.rst

```diff
@@ -693,6 +704,48 @@ Detailed field description
   legend and which defaults to the value in ``datasetId``.
 
 
+Mapping table
+-------------
+
+Mapping PEtab entity IDs to entity IDs in the model. This optional file may be
+used to reference model entities in PEtab files where the ID in the model would
+not be a valid identifier in PEtab (e.g., due to inclusion of blanks, dots, or
+other special characters).
+
+The TSV file has two mandatory columns, ``petabEntityId`` and
+``modelEntityId``. Additional columns are allowed.
+
++---------------+---------------+
+| petabEntityId | modelEntityId |
++===============+===============+
+| STRING        | STRING        |
++---------------+---------------+
+| reaction1_k1  | reaction1.k1  |
++---------------+---------------+
+
+
+Detailed field description
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- ``petabEntityId`` [STRING, NOT NULL]
+
+  A valid PEtab identifier that is not defined in any other part of the PEtab
+  problem. This identifier may be referenced in condition, measurement,
+  parameter and observable tables, but cannot be referenced in the model
+  itself.
+
+- ``modelEntityId`` [STRING, NOT NULL]
+
+  A globally unique identifier defined in the model.
```

Comment:
```suggestion
  A globally unique identifier defined in the model, *that is not a valid PEtab ID*.
```

That would make life much easier, since we won't have multiple PEtab parameter IDs mapping to the same model parameter. I don't see a good reason to support several aliases for the same entity, it will only make it more difficult to interpret the PEtab files.

CC @PEtab-dev/petab-editors

Reply:
Makes sense, but "valid PEtab ID" is a bit ambiguous to me. Maybe "that is not directly referenced in any other PEtab tables (its `petabEntityId` should be used instead)."

Reply:
> Maybe "that is not directly referenced in any other PEtab tables (its `petabEntityId` should be used instead)."

I am happy to change the wording, but my point was rather that it *cannot* be directly referenced elsewhere (due to identifier constraints), not only that it *is not* referenced elsewhere through it's original identifier.

Reply:
Ah right, then it makes sense :+1: The rest of the text about the mapping table makes this clear too. Looks good to me.